mariamihaelafainarea/assignment3/Assignment3Test

```diff
@@ -0,0 +1,225 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment3;
+
+import java.util.Iterator;
+import java.util.Set;
+import java.util.TreeSet;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author maria
+ */
+public class Assignment3Test {
+    @Test
+    public void test1(){
```

Comment:
Could you try to name those tests in a more meaningful way? So that just the name would give you some information about what is tested.

Reply:
okey.